src/components/Box/Box.jsx

```diff
@@ -0,0 +1,598 @@
+import React, {useRef, forwardRef} from "react";
+import PropTypes from "prop-types";
```

Comment:
please fix all linter issues

Reply:
+1

Reply:
@hadasfa i will require small assistant on this.